Modules/Forum/classes/class.ilObjForumAdministrationGUI.php

```diff
@@ -139,6 +140,7 @@ protected function populateForm(ilPropertyFormGUI $form): void
         $frma_set = new ilSetting('frma');
 
         $form->setValuesByArray([
+            'forum_enable_print' => (string) $this->settings->get('forum_enable_print', '0'),
```

Comment:
IMO we should cast the result to `(bool)` because it is a checkbox element, right?